### DISCUSSION TITLE
Use Shared VPC pattern with infrastructure-live

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#652] [How do I] [answered]
Do we have any documentation or tips for using the Shared VPC pattern, https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html, with the Reference Architecture/infrastructure-live? Are there other Gruntwork modules using AWS RAM? 

---

<ins datetime="2023-01-30T15:47:45Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109850">Tracked in ticket #109850</a></p>
</ins>

**Comment by zackproser** [comment, marked as answer]
We have [a guide to VPC sharing here](https://docs.gruntwork.io/guides/build-it-yourself/vpc/core-concepts/vpc-peering). 

We also have the following examples available in our [terraform-aws-vpc](https://github.com/gruntwork-io/terraform-aws-vpc) repository: 

* [VPC peering cross-accounts](https://github.com/gruntwork-io/terraform-aws-vpc/tree/main/examples/vpc-peering-cross-accounts)
* [VPC peering external](https://github.com/gruntwork-io/terraform-aws-vpc/tree/main/examples/vpc-peering-external)
* [VPC peering](https://github.com/gruntwork-io/terraform-aws-vpc/tree/main/examples/vpc-peering)

> Are there other Gruntwork modules using AWS RAM?

Please see [our KB post on determining whether or not Gruntwork offers a particular module.](https://github.com/gruntwork-io/knowledge-base/discussions/653)
